### ISSUE TITLE
IsMultiLingual error on build

### ISSUE THREAD

**Issue by hennio** [alex-shpak/hugo-book#624]
Building fails at `IsMultilingual`. My site is not multilingual btw. 

**Error** 

```
Error: error building site: render: failed to render pages: render of "taxonomy" failed: "/opt/buildhome/repo/themes/hugo-book/layouts/_default/baseof.html:50:5": execute of template failed: template: taxonomy/taxonomy.html:50:5: executing "menu" at <partial "docs/menu" .>: error calling partial: "/opt/buildhome/repo/themes/hugo-book/layouts/partials/docs/menu.html:4:6": execute of template failed: template: partials/docs/menu.html:4:6: executing "partials/docs/menu.html" at <hugo>: can't evaluate field IsMultilingual in type interface {}
```

**hugo.toml**

```
baseURL = 'https://xxxxxxxxxx.pages.dev'
languageCode = 'en-us'
title = 'My shared pages'
theme = 'hugo-book'
```

**Comment by kyrylo** [comment]
I have the same issue. Locally, it builds, but when I deploy with Cloudflare Pages, I see this error.

**Comment by chmac** [comment]
It could be that the version of hugo you're using to build is too old. I had this same issue, and eventually realised that debian (on my build docker container) is shipping an old version of hugo.

**Comment by chmac** [comment]
Actually, sorry, that was a mistake. I did update hugo, and thought the issue was fixed, but it is not. My workaround was to fork the repo, remove the offending piece from the `menu.html` and switch my git submodule to my fork instead. My fork is [here](https://github.com/chmac/hugo-book) if it helps anyone, but be advised it will get out of date as I won't copy any updates from upstream. Hopefully this issue gets fixed and I can kill my fork.
